puffin_egui/src/flamegraph.rs

```diff
@@ -845,4 +906,14 @@ fn paint_thread_info(info: &Info, thread: &ThreadInfo, pos: Pos2) {
     info.painter
         .rect_filled(rect.expand(2.0), 0.0, Rgba::from_black_alpha(0.5));
     info.painter.galley(rect.min, galley);
```

Comment:
You can add a nice hover-effect here:

```suggestion
    let text_color = if is_hovered { Rgba::WHITE } else { Rgba::from_white_alpha(0.9) };
    info.painter.galley(rect.min, galley, text_color);
```

(if you move up the `let is_hovered` a few lines.)

This will override the color given in `layout_no_wrap`, so that can be changed to anything, e.g. `Color32::TEMPORARY_COLOR`

Reply:
Nice, also added slight background alfa hover effect as hovering text color change was barely noticable.

https://user-images.githubusercontent.com/19969910/193409853-31d93251-8339-4ff7-8193-82047577acdc.mp4